index.html

```diff
@@ -67,5 +67,130 @@ <h5 class="small-text">I can help you build a product , feature or website Look
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+           </p>
+           <ul class="tags">
+            <li class="tag-item">CSS</li>
+            <li class="tag-item">HTML</li>
+            <li class="tag-item">Bootstrap</li>
+            <li class="tag-item">Ruby</li>
+           </ul>
+           <button type="button">See Project</button>
+        </div>
+        <div class="works-grid">
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+           </div> 
+           <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+        </div>
+        <div class="work-item">
+            <div class="work-item-content">
+                <h3>Multi-Post Stories</h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul class="tags">
+                    <li class="tag-item">CSS</li>
+                    <li class="tag-item">HTML</li>
+                    <li class="tag-item">Bootstrap</li>
+                    <li class="tag-item">Ruby</li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+            </div>
+            </div>
+    </section>
+
+    <section class="about-page">
```

Comment:
- [ ] Kindly use the correct background image for this section as provided by the chosen Figma design:

**This is how your section looks like:**

![image](https://user-images.githubusercontent.com/86318284/185757035-8ccf7ada-39a5-4f4b-b764-f05e6c2d6615.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/185757056-12b8e73d-1239-4011-8574-4307238b9d39.png)